apps/src/code-studio/components/progress/ProgressDot.jsx

```diff
@@ -202,8 +215,8 @@ export const ProgressDot = Radium(React.createClass({
 
     // fa-list-ol is used only by our redesigned dots, but we don't want to use
     // it here
-    if (level.icon && level.icon !== 'fa-list-ol') {
-      return 'fa ' + level.icon;
+    if (level.icon) {
+      return iconClassFromIconType[level.icon];
```

Comment:
You used to return `''` empty string in the `fa-list-ol` case; now you return `undefined`.  Intentional?